go.mod

```diff
@@ -1,17 +1,19 @@
 module chainguard.dev/melange
 
-go 1.23.4
+go 1.24.0
+
+toolchain go1.24.1
 
 require (
-	chainguard.dev/apko v0.25.4
+	chainguard.dev/apko v0.25.6-0.20250328195433-4c14aaa28144
 	cloud.google.com/go/storage v1.51.0
```

Comment:
we ought to cut new apko release; and bump melange to it; regardless of this pr.

such that ideally this pr doesn't touch upgrades of apko.